AgateDecorator/AgateDecorator.js

```diff
@@ -1,9 +1,9 @@
 /**
- * Applies Agate specific styling and behaviors.
+ * {@link agate/AgateDecorator} will be removed in 2.0.0. Use {@link agate/ThemeDecorator} instead.
  *
  * @module agate/AgateDecorator
  * @exports AgateDecorator
- * @deprecated Will be removed in 2.0.0. Use {@link agate/ThemeDecorator} instead.
+ * @deprecated
```

Comment:
I'm not should this part it right.  @ybsung  Please check when you merge it.

Reply:
Could you share why did you change the JS Doc here for the deprecated?

Reply:
That's because there is no reason to @deprecated tag with the sentance there. 
`@deprecated Will be removed in 2.0.0. Use {@link agate/ThemeDecorator} instead.`
' Will be removed in 2.0.0. Use {@link agate/ThemeDecorator} instead.'
Both are same results on the doc page